android/src/main/java/ie/gov/tracing/Tracing.kt

```diff
@@ -695,12 +716,12 @@ object Tracing {
             val result: WritableMap = Arguments.createMap()
             val typeData: WritableArray = Arguments.createArray()
             val isPaused = SharedPrefs.getBoolean("servicePaused", context)
-            if (isPaused) {
+            if (doesSupportENS && isPaused) {
                 exposureDisabledReason = "paused"
             }
 
             try {
-                if (!isBluetoothAvailable()) {
+                if (doesSupportENS && !isBluetoothAvailable()) {
                     exposureStatus = EXPOSURE_STATUS_DISABLED
                     exposureDisabledReason = "bluetooth"
                 } else if (exposureDisabledReason == "bluetooth") {
```

Comment:
The above change with this `else if` looks like, if `!doesSupportENS` and bluetooth is still unavailable, `exposureDisabledReason` will change from `"bluetooth"` to `""`?

If `!doesSupportENS`, should we just return `getExposureStatus` early with an appropriate status and skip the other checks and variable changes?

Reply:
the else if would only change if exposureDisabledReason is set to "bluetooth" which would make sense to clear even if doesSupportENS is false as the reason cannot be bluetooth. 
If doesSupportENS is false then the status is set elsewhere so I don't want to duplicate setting it again

Reply:
Makes sense. @colmharte  Should we therefore also check `doesSupportENS` in `BleStatusReceiver`'s `onReceive`, to prevent that setting `EXPOSURE_STATUS_DISABLED` and `"bluetooth"` while `!doesSupportENS`?

Reply:
Yes good call, I'll add this